src/store/whiteboarding/fabric-utils.ts

```diff
@@ -445,6 +450,40 @@ const $_addCanvasListeners = (state: any) => {
     $_enableCanvasElements(true)
   })
 
+  p.$canvas.on('mouse:wheel', (opt: any) => {
+    $_zoom(opt.e.deltaY, {x: opt.e.offsetX, y: opt.e.offsetY})
+    opt.e.preventDefault()
+    opt.e.stopPropagation()
+  })
+
+  p.$canvas.on('mouse:down', function(opt) {
+    const evt = opt.e
+    if (evt.altKey === true) {
+      this.isDragging = true
+      this.selection = false
+      this.lastPosX = evt.clientX
+      this.lastPosY = evt.clientY
+    }
+  })
+
+  p.$canvas.on('mouse:move', function(opt) {
+    if (this.isDragging) {
+      const e = opt.e
+      const vpt = this.viewportTransform
+      vpt[4] += e.clientX - this.lastPosX
+      vpt[5] += e.clientY - this.lastPosY
+      this.requestRenderAll()
+      this.lastPosX = e.clientX
+      this.lastPosY = e.clientY
```

Comment:
Use alt-key (or option-key) plus `mouse:move` to pan the canvas. Ot zoom in and out, use (1) mouse:wheel or (2) buttons in toolbar.